Assets/Scripts/KuriTextManager.cs

```diff
@@ -91,9 +119,17 @@ public enum PRIORITY {
                 audioSource.Play();
                 foreach (char letter in processTuple.text) {
                     kuriTextMesh.text += letter;
+                    //flickers light at each letter
+                    ToggleGlow();
                     yield return new WaitForSeconds(textTypingTime);
                 }
                 audioSource.Pause();
+                //turns light off
```

Comment:
If you need this, maybe you make "ToggleGlow" with an argument of True of False (then rename it to something like "SetGlow" or something like that